syntax="proto3";

package version;

enum Version {
    BEGIN           = 0;
    PROTO_VER       = 1;    //当前协议版本
}